tabs-tabs-tabs/tabs.html

```diff
@@ -33,6 +33,13 @@
     <a href="#" id="tabs-default-zoom">Reset zoom</a><br>
     <a href="#" id="tabs-decrease-zoom">Zoom out</a>
 
+    <div class="panel-section-separator"></div>
+
+    <div class="switch-tabs">
+      <p>Switch tabs</p>
```

Comment:
The term switch to me means swap two things around, could just be me. I wonder if "Active tab" or "Switch to tab" would make it clearer.